src/routes/cart/+page.svelte

```diff
@@ -1,40 +1,75 @@
 <script lang="ts">
+	import { enhance } from '$app/forms';
 	import type { PageData } from './$types';
 
 	export let data: PageData;
+
+	const fmt = new Intl.NumberFormat('pt-BR', {
+		style: 'currency',
+		currency: 'BRL'
+	});
 </script>
 
 <table class="table">
 	<thead>
 		<tr>
 			<th>Produto</th>
 			<th>Categoria</th>
-			<th>Quantidade</th>
 			<th>Preço</th>
+			<th>Quantidade</th>
 			<th>Total</th>
 		</tr>
 	</thead>
 	<tbody>
-		{#each data.sales as sale}
+		{#each data.sales.toSorted( (a, b) => (String(a.products?.name) > String(b.products?.name) ? 1 : -1) ) as sale}
```

Comment:
toSorted method isn't avaliable in node v18 (node version of vercel)
```suggestion
		{#each data.sales as sale}
```
this will remove the alphabetic sorting of sales